src/ci/release.sh

```diff
@@ -26,7 +26,7 @@ TAG_NAME="okta-sdk-root-${NEW_VERSION}" # default release plugin tag format
 
 # TODO: we must use our local maven settings file as this script is NOT ready for triggered by travis
```

Comment:
Is this TODO related to the change you made? i.e. does this still apply?

Reply:
Still applies, still a little work to do on that front